pimpmycause/pimpmycause/templates/registration/password_change_done.html

```diff
@@ -0,0 +1,16 @@
+{% extends "layout.html" %}
+
+{% block title %} Change Password {% endblock %}
+
+{% load static %}
+
+{% block main %}
+  <div class="content-wrap">
```

Comment:
Can you please add the divs and wrappers that contain in similar to other pages, foe example: https://github.com/womenhackfornonprofits/pimp-my-cause-django/blob/master/pimpmycause/pimpmycause/templates/profiles/register.html#L8-L13